CHANGELOG.rst

```diff
@@ -12,6 +12,7 @@ Depreciated
 
 Features
 --------
+- re-written ``metrics`` and ``comparisons`` based on classes to allow user metrics and user comparisons. (:pr:`2xx`) `Aaron Spring`_.
```

Comment:
Can update this to PR 268.

Reply:
For consistency, it would be good to say "Rewrote". I'm wondering if it would also be a good practice to include a quick code snippet below this to demonstrate the idea, as they do in http://xarray.pydata.org/en/stable/whats-new.html sometimes.

Reply:
maybe link to the new documentation sections user-defined metrics/comparisons?

Reply:
That sounds good. I think it's just good to take the extra time (now and in the future) for the CHANGELOG to have examples and links so if someone looks through it, it's more clear what these updates mean for them as a user.